File: backend/trivia-data/src/main/java/com/delacruz/trivia/model/Player.java
package com.ectrvia.trivia.model;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class Player {
    private Long id;
    private Long roomId;
    private String roomCode;
    private String nickname;
    private Boolean isHost;
    private Boolean isProxyHost;
    private Integer joinOrder;
    private Integer totalScore;
    private Integer currentStreak;
    private Boolean isConnected;
    private LocalDateTime joinedAt;
}

File: ECTriviaApp/app/src/main/java/com/delacruz/ectrivia/data/model/Player.kt
package com.ectrvia.ectrivia.data.model

data class Player(
    val id: Long,
    val nickname: String,
    val isHost: Boolean,
    val isProxyHost: Boolean,
    val totalScore: Int,
    val currentStreak: Int,
    val isConnected: Boolean
)
